### PR TITLE
chore: run all tests with code coverage

### LINKED ISSUE #1455: Fix code coverage check
Our code coverage check is failing, either for missing credentials or because our test coverage has decreased.

We should match the AlloyDB Java Connector's code coverage check: https://github.com/GoogleCloudPlatform/alloydb-java-connector/blob/main/.kokoro/check_coverage.sh.

### PR DESCRIPTION
**Pull request by ttosta-google**
Fixes #1455

### CODE REVIEW

**Review by hessjcg — COMMENTED** [review]
LGTM.

File: .kokoro/check_coverage.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# limitations under the License.
+
+# `-e` enables the script to automatically fail when a command fails
+set -e
+
+export CUR_COVER=$(cat core/target/site/jacoco/index.html | grep -o 'Total[^%]*' | sed 's/<.*>//; s/Total//')
+echo "Current Coverage is $CUR_COVER%"
+if [ "$CUR_COVER" -lt 75  ]; then
+  exit 1;
+fi
```
Comment (hessjcg) [inline .kokoro/check_coverage.sh:23]: Needs a newline at the end of the file.

Comment (ttosta-google) [reply .kokoro/check_coverage.sh]: added. Thanks!

**Review by hessjcg — APPROVED** [review]
LGTM

**Pull request merged**